src/Nethermind/Nethermind.Clique.Test/CliqueBlockProducerTests.cs

```diff
@@ -1,5 +1,18 @@
-// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
-// SPDX-License-Identifier: LGPL-3.0-only
+//  Copyright (c) 2021 Demerzel Solutions Limited
```

Comment:
It should be SPDX file, right?

Reply:
How do you judge if it should be SPDX or other?